back/src/bsffs/bsffs.types.graphql

```diff
@@ -0,0 +1,197 @@
+}
+enum BsffPackagingType {
+  BOUTEILLE
+}
+
+type BsffWaste {
+  "Code déchet."
+  code: String!
+  "Description du déchet, permet de le qualifier de façon plus précise."
+  description: String!
+  "Mention ADR."
+  adr: String!
+}
+
+type BsffQuantity {
+  "Poids total du déchet en kilos."
+  kilos: Int!
+  "Si il s'agit d'une estimation ou d'un poids réel."
+  isEstimate: Boolean!
+}
+
+type BsffTransporter {
+  "Entreprise responsable du transport du déchet."
+  company: FormCompany!
+  "Récépissé du transporteur, à moins d'être exempté."
+  recepisse: BsffTransporterRecepisse
+  "Déclaration du transporteur lors de l'enlèvement auprès de l'émetteur."
+  transport: BsffTransport
+}
+type BsffTransporterRecepisse {
+  "Numéro du récépissé."
+  number: String!
+  "Département auquel est lié le récépissé."
+  department: String!
+  "Date limite de validité du récépissé."
+  validityLimit: DateTime!
+}
+type BsffTransport {
+  "Mode de transport utilisé."
+  mode: TransportMode!
+  "Signature du transporteur lors de l'enlèvement auprès de l'émetteur."
+  signature: Signature!
+}
+
+type BsffDestination {
+  "Entreprise réceptionant le déchet."
+  company: FormCompany!
+  "Déclaration de réception du déchet."
+  reception: BsffReception
+  "Déclaration de traitement du déchet."
+  operation: BsffOperation
+  "Opération de traitement prévu initiallement."
+  plannedOperation: BsffPlannedOperation!
+  "Numéro CAP."
+  cap: String
+}
+type BsffReception {
+  "Date de réception du déchet."
+  date: DateTime!
+  "Quantité totale du déchet, qu'elle soit réelle ou estimée."
+  kilos: Int!
+  "En cas de refus, le motif."
+  refusal: String
+  "Signature de la destination lors de l'acceptation ou du refus du déchet."
+  signature: Signature
+}
+interface IBsffOperation {
+  "Code de l'opération de traitement."
+  code: BsffOperationCode
+  "Qualification plus précise du type d'opération réalisée."
+  qualification: BsffOperationQualification!
+}
+type BsffOperation implements IBsffOperation {
+  "Code de l'opération de traitement."
+  code: BsffOperationCode
+  "Qualification plus précise du type d'opération réalisée."
+  qualification: BsffOperationQualification!
+  "Signature de la destination lors du traitement."
+  signature: Signature
+}
+type BsffPlannedOperation implements IBsffOperation {
+  "Code de l'opération de traitement prévu."
+  code: BsffOperationCode
+  "Qualification plus précise du type d'opération prévu."
+  qualification: BsffOperationQualification!
+}
+"Liste des codes de traitement possible."
+enum BsffOperationCode {
+  R2
+  R12
+  D10
+  D13
+  D14
+}
+"""
+Liste des qualifications de traitement possible.
+Attention, certaines combinaisons de code et qualification ne sont pas possibles.
+Par exemple, seul le code D 10 peut être associé à une incinération.
+"""
+enum BsffOperationQualification {
+  RECUPERATION_REGENERATION
+  INCINERATION
+  REGROUPEMENT
+  RECONDITIONNEMENT
+  REEXPEDITION
+}
+
+type BsffFicheIntervention {
```

Comment:
Idem ici je ne fait pas trop le lien avec le CERFA N°15497*02  "Fiche d'intervention / Bordereau de suivi de déchets dangereux". En effet on ne retrouve pas dans ce type tous les champs coloriés en vert sur le document papier: détecteur manuel de fuite, présence d'un système de détecteur de fuites, fuites constatées lors du contrôle d'étanchéité, etc.

Reply:
C'est des informations techniques qui dépassent le cadre du traitement du déchet qu'on aimerait dissocier. L'idée c'est d'avoir ces informations sur une fiche d'intervention qu'on ne récupère pas, et récupérer les informations liés au déchet. C'est pour ça que le modèle `BsffFicheIntervention` est simplifié, il ne contient que ce qui a un intérêt pour nous.